docs/content/api/index.ngdoc

```diff
@@ -5,7 +5,7 @@
 # AngularJS API Docs
 
 <div class="alert alert-warning">
-**AngularJS will be moving to Long Term Support (LTS) mode on July 1st 2018.**: [Find out more](misc/version-support-status).
+**On July 1, 2018 AngularJS entered a 3 year Long Term Support period..**: [Find out more](misc/version-support-status).
```

Comment:
Double dots after period 😱 In fact non of the dots is needed, since they are followed by `:`. Can you, please, remove them both?

Also, for the sake of consistency (with the README 😳), lets put the closing `**` after the `:` 😇

Reply:
updated. 👍